data/newMigrations/006-service_types.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('service_types', (tbl) => {
+    tbl.increments('service_type_id').primary();
+    tbl.foreign('program_id').references('programs.program_id');
+    tbl.string('service_type_name', 255);
+    tbl.text('service_type_description');
+    tbl.jsonb('service_type_entry_model');
+    table.timestamps(true, true);
```

Comment:
This should be "tbl.timestamps" rather than "table.timestamps"

Reply:
Ah, you're right